### DISCUSSION TITLE
Possible rework idea for Spineling Genes and Spineling's Spike

### DISCUSSION THREAD

**Discussion by Stallfighter** [FakeFishGames/Barotrauma#10885] [Ideas and suggestions]
From 3 genes that actually affect player model directly - raptor, husk (kinda) and spineling, the latter is the most underwhelming. 

Raptor genes are also not the greatest but they do actually have usage, and it is similar to how its respective monster works - tearing through any door or hatch on its way.

Spineling ones are supposed to do "ranged damage" in which they utterly fail - there are little to no alpha strike and no DPM at all considering how slow they regenerate on your spine, and its hard to aim for no reward.

But I have an idea, which keeps the spirit of the moster and to some degree its tactics - make it into the "free" harpoon gun ammunition.

With these genes equipped, player character receives 1 Spineling's Spike once in a while, repurposed from useless "sword" item to improved penetration but lower damage harpoon. How it is happening exactly game-play wise can be discussed (Is it spawns in the player inventory? In hands? A special brand-new slot for it? Or maybe, you receive them at the start of the round in some quantity, similar how Blood Donor talent works).

This addresses both the issue of usability of spines on your back, usefulness of drops of the spineling itself, and brings something interesting to the game.

**Comment by ubertpendragon** [comment]
You mean like this? 👀 

https://user-images.githubusercontent.com/73229309/211128629-1cb94760-a6a8-4b9a-a5e9-e4b44ac5c203.mp4

**Reply by Stallfighter** [reply]
> You mean like this?

Well, while this might work you couldn't resupply yourself in the water, because they sink too fast. I guess if you mod neutral buoyancy as well it can be a good framework for devs.
